### PR TITLE
airtable api in a component

### PR DESCRIPTION
**Pull request by harshbrahmxatriya**
Airtable api in a component named airtable-demo.js in the components folder
The page is cars.vue in the pages folder
This is an initial commit to share the airtable api connection

### CODE REVIEW

File: pages/cars.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <li v-for="i in list">{{ i }}</li>
```
Comment (razbakov) [inline pages/cars.vue:3]: See how to use `v-for` [here](https://vuejs.org/api/built-in-directives.html#v-for)
You need to add `:key`

**Review by razbakov — CHANGES_REQUESTED** [review]

**Comment by razbakov** [comment]
@harshbrahmxatriya your deploy failed. See logs from Netlify
@amanuela97 would you like to help Harsh to fix it?

File: pages/cars.vue (hunk 1 above)
Comment (harshbrahmxatriya) [reply pages/cars.vue]: Okay

File: pages/cars.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+</template>
+<script>
+export default {
+  name: 'Airtable',
+  data() {
+    return {
+      list: [],
+    }
+  },
+  async created() {
```
Comment (razbakov) [inline pages/cars.vue:29]: `created` event can't be async. Try to use `fetch` from [nuxt lifecycle](https://nuxtjs.org/docs/concepts/nuxt-lifecycle/) or `mounted` from [vue lifecycle](https://vuejs.org/api/options-lifecycle.html)

Also check errors in browser console.

File: server/api/adb.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { getRecords } from "~~/components/airtable-demo"
+
+
+export default async (req, res) => {
+    try{
```
Comment (razbakov) [inline server/api/adb.js:5]: Please activate prettier in VSCode. Code is not properly formatted.

File: components/airtable-demo.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+);
+
+const table = base('Tasks');    
+const getRecords = async () => {
+    const records = await table.select().firstPage();
+    console.log(records);
+    return records
+}
+
+export{getRecords}
```
Comment (razbakov) [inline components/airtable-demo.js:14]: Please always add a blank line in the end of each file

**Review by razbakov — CHANGES_REQUESTED** [review]

File: components/airtable-demo.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import Airtable from 'airtable'
+
+var base = new Airtable({ apiKey: "keyQ54SEzABIZEm9F" }).base(
+  "appv6M4U4ckXcHzhU"
```
Comment (razbakov) [inline components/airtable-demo.js:4]: Don't export API keys in the source code. Use environment variables.

File: components/airtable-demo.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+);
+
+const table = base('Tasks');    
+const getRecords = async () => {
+    const records = await table.select().firstPage();
+    console.log(records);
```
Comment (razbakov) [inline components/airtable-demo.js:10]: Please leave debug code locally, don't commit `console.log` statements.

**Review by razbakov — CHANGES_REQUESTED** [review]

File: pages/cars.vue (hunk 2 above)
Comment (harshbrahmxatriya) [reply pages/cars.vue:29]: Okay, let me see

File: server/api/adb.js (hunk 3 above)
Comment (harshbrahmxatriya) [reply server/api/adb.js:5]: I have it activated

File: components/airtable-demo.js (hunk 4 above)
Comment (harshbrahmxatriya) [reply components/airtable-demo.js:14]: okay

File: components/airtable-demo.js (hunk 5 above)
Comment (harshbrahmxatriya) [reply components/airtable-demo.js:4]: Will see to it online, will ask you if you if reqd.

File: components/airtable-demo.js (hunk 6 above)
Comment (harshbrahmxatriya) [reply components/airtable-demo.js:10]: Oh Okay

File: pages/cars.vue (hunk 2 above)
Comment (harshbrahmxatriya) [reply pages/cars.vue:29]: I tried to use the fetch and mounted one after the another, both of them were not able to render the list

**Pull request closed without merging**